nucypher/characters/lawful.py

```diff
@@ -282,10 +282,10 @@ def generate_policy_parameters(self,
         if self.federated_only:
             if not expiration:
                 raise TypeError("For a federated policy, you must specify expiration (payment_periods don't count).")
-            if expiration <= maya.now():
+            if expiration <= maya.now().epoch:
```

Comment:
`expiration` is of type `MayaDT` in the method signature, should it be `int` instead?